src/algebra/module/equiv.lean

```diff
@@ -539,6 +539,18 @@ end add_comm_monoid
 
 end linear_equiv
 
+namespace linear_equiv
+variables {𝕜 α β : Type*} [semiring 𝕜] [add_comm_group α] [add_comm_group β] [module 𝕜 α]
+  [module 𝕜 β]
+
+@[simp]
+lemma add_subgroup_map_symm_apply (e : α ≃ₗ[𝕜] β) {L : add_subgroup α} {g : L.map (e : α →+ β)} :
+  (add_equiv.add_subgroup_map e L).symm g =
```

Comment:
Ouch, this lemma demonstrates one of the dangers of using `add_equiv_class` everywhere. This scales really poorly; what about alg_equiv, linear_isometry_equiv, continuous_linear_equiv, ...

Reply:
Yeah, this is why @Vierkantor's work is so important. But I don't think the answer is to not use `add_equiv_class`. It was made for such situations and it does improve my code in #2819.

Reply:
Can we compromise on splitting this PR into two; leaving this half for the `add_equiv_class` generalization, and spinning out another PR for the more basic lemmas and docstrings?

Reply:
@YaelDillies I'll mark this `awaiting-author`, since it looks like you want to split this PR.